config/artsey.dtsi

```diff
@@ -142,8 +142,14 @@
  * Define custom layer
  *
  * You can override this layer by copy/pasting the into your keymap *before* including the artsey.dtsi file
- * Once added to your keymap you can adjust as you see fit
+ * Once added to your keymap you can adjust as you see fit.
+ * There is also the TIMEOUT_COMBO_CUSTOM_LAYER variable that can be tweaked for the timeout on toggling the
+ *    custom layer
 *****************************************/
+#if !defined TIMEOUT_COMBO_CUSTOM_LAYER
+#define TIMEOUT_COMBO_CUSTOM_LAYER 175
+#endif
+
```

Comment:
This should be defined with the other combo timeouts and initialized the same as the others.

Reply:
Agreed. I'll change that.